### DISCUSSION TITLE
Error: module build failed

### DISCUSSION THREAD

**Discussion by nashvegasnate** [nss-evening-cohort-14/evening-client-side#194] [Help Me!] [answered]
Your Name:
Nathan Keith

Have you searched the other help tickets? 
YES - #135  #16 #98

Describe your issue:
My group has already merged their portions of this particular stage in the project. When I pull down the development branch to test it in conjunction with my code, I get the above error message and a blank screen.

List any error messages that appear:
![image](https://user-images.githubusercontent.com/76710923/111684483-c3b32680-87f4-11eb-934d-04340b1dc8bc.png)

What have you googled? 
https://github.com/sass/node-sass/issues/1579
https://github.com/jhipster/generator-jhipster/issues/6032
https://stackoverflow.com/questions/43994097/ionic-error-module-build-failed-error-enoent-no-such-file-or-directory

What have you tried? 
I have tried re-running "npm install" which seems to repopulate the files in my local node_modules directory that my console is telling me do not exist. After comparing all the directories in my local to those in the group project, it appears that while those files do exist in my local directories, they do not exist in our group development branch of the project. Thus, it cannot make the connection when attempting to merge.
I also tried "npm i gulp-sass --save-dev".
Neither seem to work.

Link to the repo: https://github.com/nss-evening-cohort-14/pan-am-group-project/tree/development
Since you can't access those node_modules files, here are screenshots: 
![image](https://user-images.githubusercontent.com/76710923/111692906-fb729c00-87fd-11eb-882a-2dfe5562a3cb.png)
Development branch above. My personal branch below. Notice different files.
![image](https://user-images.githubusercontent.com/76710923/111693379-8a7fb400-87fe-11eb-9c40-385b7586e95e.png)

**Comment by TrinityChristiana** [comment, marked as answer]
What's your node version. You can run this command in your terminal to find out.
```
node -v
```

**Reply by drteresavasquez** [reply]
https://github.com/nss-evening-cohort-14/evening-client-side/discussions/98

**Reply by nashvegasnate** [reply]
I have v14.15.4
I entered the command "npm install node-sass" earlier today and it seemed to work with all of our collective code. I hadn't updated the issue ticket yet since we hadn't yet been approved to merge our development branch to the main branch. 

I see that there is a solution suggesting I enter "npm remove gulp-sass", however, I am reluctant to try. Should I be concerned or just try it?

**Reply by TrinityChristiana** [reply]
That looks right. 🤔  What's the response to this command while in your team's repo?
```
npm node-sass -v
```

**Reply by nashvegasnate** [reply]
Here is the response I get using "npm node-sass -v" (prior to the uninstall command): 6.14.10

**Reply by TrinityChristiana** [reply]
Try deleting your node_modules folder, removing that `gulp-sass` line from the package.json, then re-run npm install and see if that does anything.

**Reply by nashvegasnate** [reply]
Looks like everything has successfully compiled. My DOM looks correct. Terminal shows the following:

_added 1200 packages from 571 contributors and audited 1200 packages in 26.444s
72 packages are looking for funding
  run `npm fund` for details
found 0 vulnerabilities_

We are going to try and merge this with our development branch and insure it works properly with Netlify deploy.

**Reply by TrinityChristiana** [reply]
Great! If it merges can you add the solution to the problem as a comment outside of this reply thread?

**Reply by BenJarrett** [reply]
Now, we are going to merge nathan's code and our dev branch locally and then push it back up to our dev branch to see if this has fixed our Netlify issues

**Reply by BenJarrett** [reply]
It didn't fix it the netlify issue, still getting an API key is invalid. We have looked at this over and over, Referred to the other tickets about invalid api keys #193 , which was ours and we fixed by taking out the commas in the .env file. But everything matches. Tried to serve and re serve like in #125 , We are going to double-check our Netlify environmental variables again

**Reply by BenJarrett** [reply]
Our environmental variables look correct. I'm not gonna lie, we are completely stuck on this. 

We fixed our API key yesterday with this issue by removing commas. 
I'm completely stuck atm

**Reply by drteresavasquez** [reply]
That's a different issue than this one. Either search for a similar ticket or create a new one please 🤓

**Reply by BenJarrett** [reply]
Got it! I'll start a new one, now 😃

**Reply by BenJarrett** [reply]
I'm gonna comment on #125 since it's a very similar issue. Let me know if this is okay or not. I'll make a new one if that isn't okay
